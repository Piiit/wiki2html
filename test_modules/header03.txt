~~NOTOC~~
= header =
== header 2 ==
