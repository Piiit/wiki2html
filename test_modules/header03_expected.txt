<!doctype html>
<html>
<head>
<title>header</title>
</head>
<body>
<h1 id='header_0'>header</h1>
<h2 id='header_1'>header 2</h2>
</body>
</html>

